### PR TITLE
fixErrorMessages : Fixing error messages

### PR DESCRIPTION
**Pull request by neeradanelxsi**

### CODE REVIEW

File: cypress/plugins/testDataProcessor.js [hunk 1]
```diff
@@ -102,45 +102,22 @@ function processSetResponseJson(setResponseJsonData) {
 }
 
 /**
- *  @function processErrorContentJson
- *  processErrorContentJson function will perform following operations
+ *  @function getErrorContentObjectJson
+ *  getErrorContentObjectJson function will perform following operations
  *  - Iterate over each key in the provided JSON
```
Comment (jbigel) [inline cypress/plugins/testDataProcessor.js:107]: This description no longer reflects what is happening in the function.

File: cypress/plugins/testDataProcessor.md [hunk 2]
```diff
@@ -44,9 +44,9 @@ Response
 
 - {'abc': {'method': 'method_name', 'validationJsonPath': 'result', 'content': true}}
 ```
-
-### > processErrorContentJson
-processErrorContentJson function will perform following operations
+getErrorContentObjectJson
+### > 
+getErrorContentObjectJson function will perform following operations
```
Comment (jbigel) [inline cypress/plugins/testDataProcessor.md:49]: same as above.

**Review by jbigel — APPROVED** [review]
Approving to unblock reports but would like to see changes.

Ideally I'd like to see FCS define the different error types and config modules can set what error codes correspond to the various types. Then in the testDataProcessor we can fetch the error codes from the config mod and plug it into our expected error types.

File: cypress/plugins/testDataProcessor.js (hunk 1 above)
Comment (Abhishk123) [reply cypress/plugins/testDataProcessor.js]: addressed the comment

File: cypress/plugins/testDataProcessor.md (hunk 2 above)
Comment (Abhishk123) [reply cypress/plugins/testDataProcessor.md]: addressed the comment

**Review by jbigel — APPROVED** [review]

**Pull request merged**